R/api.R

```diff
@@ -411,6 +412,24 @@ endpoint_run_metadata <- function(runner) {
  )
 }
 
+target_workflow_missing_dependencies <- function(path, body) {
+  body <- jsonlite::fromJSON(body)
+  tasks <- body$tasks
+  workflow_missing_dependencies(path, tasks)
+}
+
+endpoint_workflow_missing_dependencies <- function(path) {
```

Comment:
What is your thoughts here of making this a function of `path` rather than a method of the runner?

Reply:
Trying to stop `runner` becoming some massive object which does everything only going through the `runner` when the function interacts with the queue.